P202.Training.Data/Security/RoleRepository.cs

```diff
@@ -0,0 +1,64 @@
+﻿using NHibernate;
+using P202.Training.Data.Entities;
+using P202.Training.Data.Helper;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace P202.Training.Data.Security
+{
+    public class RoleRepository
+    {
+        public void RoleCreate(Role role)
+        {
+            using (var session = NHibernateHelper.OpenSession())
+            {
+                using (var transaction = session.BeginTransaction())
+                {
+                    session.Save(role);
+                    transaction.Commit();
+                }
+
+            }
+
+        }
+
+        public IList<Role> RoleGetAll()
+        {
+
+            IList<Role> roles = new List<Role>();
```

Comment:
Minor: I think you can use `var roles = new List<Role>()`

Reply:
i use var